openshift/s2i/nginx-runtime/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use the offical nginx (based on debian)
-FROM nginx:stable
+FROM docker.io/nginx:stable
```

Comment:
Updated in an attempt to mitigate docker pull rate issue.

Reply:
Isn't there issues if we pull directly from docker.io?  Since all requests come through the same IP people are getting timeouts, as Docker limits the number of requests allowed.  I think this will lead to issues.

Reply:
yeah, just using nginx:stable also pulls from docker.io.

see the *Docker Auth* section of this article

not sure if the above actually resolves the issue however, as the docker.io part is in the dockerfile not the build config. may need to cache the nginx:stable issue instead.